Exercises/functions/functions.js

```diff
@@ -1,3 +1,32 @@
+// 1. Create a function declaration, function expression, lambda function, and using function object.
+// function declaration
+function functionDeclaration() {}
+// function expression
+const funcExpression = function () {};
+// lambda function
+const funcArrow = (param) => {
+  return 1;
+};
+const funcArrow1 = (param) => 1;
+// function object
+/*eslint-disable */
+const funcObject = new Function('a', 'b', 'return a * b');
+console.log(funcObject(4,6));
+
+// 2. Get parameters amount of function `takeMyParametersAmount`.
+
 function takeMyParametersAmount(a, b, c) {
-  return null;
+  return a + b + c;
 }
+
+// 2. Create a function and log all passed parameters to it. Check the `length` of this function.
+
+function myNewFunc(a, b, c, d, r) {}
+console.log(myNewFunc.length);
```

Comment:
Here you need to log all params that was passed to function, not just the length. So even if your function will be called with 100 parameters they all should be logged.